packages/components/src/Filter/FilterButton/FilterButton/FilterButton.module.css

```
@@ -0,0 +1,28 @@
+.filterButton {
+  gap: var(--spacing-xs);
+  justify-content: space-between;
+}
+
+/* keep max-width value in sync with other places using this container */
+@container filter-bar-wrapper (max-width: 767px) {
+  .filterButton {
+    flex-grow: 1;
+  }
+}
+
+.labelContainer {
+  display: inline-flex;
+  flex-wrap: wrap;
+}
+
+.hasSelectedValues {
+  font-weight: bold;
+}
+
+.labelSeparator {
+  margin-inline-end: var(--spacing-xs);
```

Since we're updating, might as well use the new variables :)
```suggestion
  gap: var(--spacing-6);
```